phabricator_tasks/phabricator_patched/_compat.py

```diff
@@ -0,0 +1,26 @@
+import sys
+
+PY3 = sys.version_info[0] >= 3
+
+try:
+    from collections.abc import MutableMapping
+except ImportError:
+    from collections import MutableMapping
+
+try:
+    from urllib import urlencode
+except ImportError:
+    from urllib.parse import urlencode
+
+if PY3:
```

Comment:
How can we end up with a worker host running Python2? We don't need compatibility with Python2 (which is dead), we need to make sure that the container image that runs this script is up to date. ;)